pkg/agent/agent_windows.go

```diff
@@ -187,3 +204,57 @@ func (i *Initializer) initHostNetworkFlows() error {
 func (i *Initializer) getTunnelPortLocalIP() net.IP {
 	return i.nodeConfig.NodeIPAddr.IP
 }
+
+// getHostRoutes returns routes which are configured on uplink interface before
+// the interface the configured as the uplink of antrea HNS network.
+// The routes will be restored on OVS bridge interface after the IP configuration
+// is moved to the OVS bridge.
+func (i *Initializer) getHostRoutes() ([]netroute.Route, error) {
+	nr := netroute.New()
+	defer nr.Exit()
+	routes, err := nr.GetNetRoutesAll()
+	if err != nil {
+		return nil, err
+	}
+	var hostRoutes []netroute.Route
+	for _, route := range routes {
+		if route.LinkIndex != i.nodeConfig.UplinkNetConfig.Index {
+			continue
+		}
+		if route.GatewayAddress.String() != i.nodeConfig.UplinkNetConfig.Gateway {
+			continue
+		}
+		// Skip IPv6 routes before we support IPv6 stack.
+		if route.DestinationSubnet.IP.To4() == nil {
+			continue
+		}
+		// Skip default route. The default route will be added automatically when
+		// configuring IP address on OVS bridge interface.
+		if route.DestinationSubnet.IP.IsUnspecified() {
+			continue
+		}
+		klog.V(4).Infof("Got host route: %v", route)
+		hostRoutes = append(hostRoutes, route)
+	}
+	return hostRoutes, nil
+}
+
+// restoreHostRoutes restores the host routes which are lost when moving the IP
+// configuration of uplink interface to the OVS bridge interface during
+// the antrea network initialize stage.
+// The backup routes are restored after the IP configuration change.
+func (i *Initializer) restoreHostRoutes() error {
```

Comment:
I feel more readable to make `getHostRoutes` and `restoreHostRoutes` symmetrical. Since `restoreHostRoutes` accesses `i.nodeConfig.UplinkNetConfig.Routes` directly, `getHostRoutes` could write the routes to it directly, and perhaps name it `saveHostRoutes`

Reply:
Thanks Quan for your advice.

Replace `getHostRoutes` function with `saveHostRoutes`. Done.